packages/flutter/lib/src/widgets/dismissible.dart

```diff
@@ -134,6 +135,12 @@ class Dismissible extends StatefulWidget {
   /// [direction] property.
   final Map<DismissDirection, double> dismissThresholds;
 
+  final Duration dismissDuration;
+  /// defines the duration for card to dismiss and come to original position
+
+  final double diagonalDisplacement;
```

Comment:
Seems specific. Is the desired feature just to be able to specify dismiss in an arbitrary vector?

Reply:
The movement is along the across the direction. Otherwise, the movement is just along the direction. It makes the widget more customisable.